### ISSUE TITLE
Missing Evolution Data, Encounter Data, EV data

### ISSUE THREAD

**Issue by Quikswitch** [ashencone/unbound#4]
First of all, thanks for making this! Even if it's not the singular definitive source, it is SUPER useful.

Second of all, if it's possible, adding evolution data, EV stat gains, and/or encounter data would be pretty useful.
Of course, some of this information is available on Bulbapedia or similar, but in some cases it is either different (different evolution levels, link stones, etc), unclear (Glacion/Leafeon evolution method), or absent (location/encounter data). Some of this is available via a community made document (see this link) [https://docs.google.com/spreadsheets/d/1LFSBZuPDtJrwAz7t6ZkJ-il4j8M3qCdaKLNe6EZdPmQ/edit#gid=0](url). That covers encounter data, and swarm data (if you want to go that far).
This Bulbapedia document contains the EV data [https://bulbapedia.bulbagarden.net/wiki/List_of_Pok%C3%A9mon_by_effort_value_yield#List_of_Pok.C3.A9mon_by_effort_value_yield](url).
And the evolution data can be seen in the official download under changes from official, though this excludes non-levelup based evolutions, like Ice and Link stone evolutions. Alolan forms are particularly strange, and poorly documented. [https://www.mediafire.com/file/brvvppywnxhmsdb/Pokemon+Unbound+Official+Patch+2.0+.zip/file](url).
As a bonus, you could also include shiny sprites. These are available from the official discord, but here's a zip file with them for convenience (link expires 30 days from this post) [https://ufile.io/eejyl7ra](url)
All of this should be correct as of v2.1.1 / Nov. 2022

In short, and in order of importance, these things could be added to the Unbound Pokedex:
1. Evolution data
2. Encounter/location data
3. EV yield data
4. Shiny sprite data

Evolution data is particularly important, as the only source I can find so far is in-game with the evolution house person, or the C
I understand if this isn't much of a concern, but I thought I'd offer some options to make this a definitive source of info for Pokemon Unbound. Thanks again for making this tool!

**Comment by Quikswitch** [comment]
I've noticed that EV data is present. Feel free to ignore those comments.
